docs/en-us/dev/user_doc/guide/alert/enterprise-webexteams.md

```diff
@@ -0,0 +1,23 @@
+# WebexTeams
+
+If you need to use WebexTeams to alert, please create an alarm Instance in warning instance manage, and then choose the WebexTeams plugin. The configuration example of enterprise WebexTeams is as follows:
+
+![enterprise-wechat-plugin](/img/alert/enterprise-wechat-plugin.png)
```

Comment:
Seem it is a wrong link and link text
```suggestion
![enterprise-webexteams-plugin](/img/alert/enterprise-webexteams-plugin.png)
```

Reply:
Hi @liqingwang, could you change to webexteams img?

Reply:
Hi @zhongjiajie I am sorry, it was a mistake, I have corrected it now.